src/tiledbsoma_ml/__init__.py

```diff
@@ -19,4 +20,8 @@
 __all__ = [
     "ExperimentDataset",
     "experiment_dataloader",
+    "Chunks",
```

Comment:
I'm confused why these classes (Chunks, Partition and QueryIDs) are top-level public API.   I.e., why are they exported?  Reading the code they look like implementation details, not API we want people coding to (ie., stuff we will need to maintain b/w compat with).

Reply:
Thoughts on this?  Is this driven by docs?

I just prefer that we keep the public API as clean and simple as possible, so we don't implicitly make commits that affect b/w compat, etc.

Reply:
Sorry, thought I'd responded to this. docs is a good guess why I did this, but I'm unable to repro that now. I may have thought "arg-types of public methods must be public" (instinct from JVM days).

With some distance, I don't think anything needs to or should be public besides `ExperimentDataset`, `experiment_dataloader`, and `SamplingMethod` (argument to `ExperimentDataset.random_split`).

Latest push will make private:
- `_query_ids`
- `_io_batch_iterable`
- `_mini_batch_iterable`
- `_common`

and remove them from `tiledbsoma_ml/__init__.py`. lmk what you think!